src/box/node_name.c

```diff
@@ -15,7 +15,7 @@ node_name_is_valid_n(const char *name, size_t len)
 	const char *end = name + len;
```

Comment:
I would just update the feature description in https://github.com/tarantool/doc/issues/3466, https://github.com/tarantool/doc/issues/3467 and https://github.com/tarantool/doc/issues/3468 and add `NO_DOC` with the links.

The triple `Title` field in the docbot-request unlikely works as 'file three issues'.

Reply:
Well.. I can't update until this one is merged. But I got the idea. Let's see the doc team resolution.

Reply:
I agree with @Totktonada.
Since the entire feature is not documented yet, it's better to update the original tasks.